storage/ndb/src/kernel/vm/mt.cpp

```diff
@@ -9559,92 +9553,120 @@ mt_getPerformanceTimers(Uint32 self,
 const char *
 mt_getThreadDescription(Uint32 self)
```

Comment:
Since we've already is_XXX_thread() for each type of block thread, could we use them to simplify this function() like this:

std::string res;
if (is_recv_thread(thr_no)) {
  res += "_recv";
}
if (is_ldm_thread(thr_no)) {
  res += "_ldm";
}
if (is_tc_thread(thr_no)) {
  res += "_tc";
}
if (is_main_thread(thr_no)) {
  res += "_main";
}
if (is_rep_thread(thr_no)) {
  res += "_rep";
}
assert(!res.empty())
res.erase(res.begin());

return res;

Reply:
Looks like a great suggestion, implemented it.